assemblies/rosa-install-logging.adoc

```diff
@@ -0,0 +1,29 @@
+include::_attributes/attributes-openshift-dedicated.adoc[]
+[id="rosa-install-logging"]
+:context: rosa-install-logging
+toc::[]
+
+= Installing logging add-on services {product-title}
+
+[role="_abstract"]
+Installing the logging add-on and Amazon Web Services (AWS) CloudWatch log forwarding add-on services on {product-title} (ROSA).
+
+The AWS CloudWatch log forwarding service on ROSA has the following approximate log throughput rates (per worker node). Message rates greater than these can result in dropped log messages.
```

Comment:
Can you just make a small change here that I think will be more clear.   Remove the (per worker node) from this sentence.

And add it to the top right cell.  e.g. Maximum expected rate (messages per second, per node)

Reply:
Done.